packages/autocannon-compare-cli/test/util.test.js

```diff
@@ -8,38 +10,38 @@ import {
   Kind
 } from '../util.js'
 
-tap.test('formatBytes', async t => {
-  t.equal(formatBytes(100), '100 bytes')
-  t.equal(formatBytes(1024), '1.0 KB')
-  t.equal(formatBytes(1024000), '1000 KB')
-  t.equal(formatBytes(1073741824), '1.0 GB')
+test('formatBytes', async () => {
+  assert.strictEqual(formatBytes(100), '100 bytes')
+  assert.strictEqual(formatBytes(1024), '1.0 KB')
+  assert.strictEqual(formatBytes(1024000), '1000 KB')
+  assert.strictEqual(formatBytes(1073741824), '1.0 GB')
 })
 
-tap.test('calculateDiffPercentage', async t => {
-  t.equal(calculateDiffPercentage(1, 1), '=')
-  t.equal(calculateDiffPercentage(0, 0), '=')
-  t.equal(calculateDiffPercentage(0, 10), '-100%')
-  t.equal(calculateDiffPercentage(10, 0), '+100%')
-  t.equal(calculateDiffPercentage(10, 2), '+400%')
-  t.equal(calculateDiffPercentage(2, 10), '-80%')
-  t.equal(calculateDiffPercentage(6.5, 10), '-35%')
+test('calculateDiffPercentage', async () => {
+  assert.strictEqual(calculateDiffPercentage(1, 1), '=')
+  assert.strictEqual(calculateDiffPercentage(0, 0), '=')
+  assert.strictEqual(calculateDiffPercentage(0, 10), '-100%')
+  assert.strictEqual(calculateDiffPercentage(10, 0), '+100%')
+  assert.strictEqual(calculateDiffPercentage(10, 2), '+400%')
+  assert.strictEqual(calculateDiffPercentage(2, 10), '-80%')
+  assert.strictEqual(calculateDiffPercentage(6.5, 10), '-35%')
 })
 
-tap.test('getColor', async t => {
-  t.equal(getColor('+100%', Kind.moreIsBetter), chalk.green)
-  t.equal(getColor('-100%', Kind.moreIsBetter), chalk.red)
-  t.equal(getColor('-100%', Kind.lessIsBetter), chalk.green)
-  t.equal(getColor('+100%', Kind.lessIsBetter), chalk.red)
+test('getColor', async () => {
+  assert.strictEqual(getColor('+100%', Kind.moreIsBetter), chalk.green)
+  assert.strictEqual(getColor('-100%', Kind.moreIsBetter), chalk.red)
+  assert.strictEqual(getColor('-100%', Kind.lessIsBetter), chalk.green)
+  assert.strictEqual(getColor('+100%', Kind.lessIsBetter), chalk.red)
 })
 
-tap.test('formatValue', async t => {
-  t.test('should format generic integers', async t => {
-    t.equal(formatValue(1300, 'requests.total'), '1,300')
+test('formatValue', async t => {
+  await t.test('should format generic integers', async () => {
+    assert.strictEqual(formatValue(1300, 'requests.total'), '1,300')
   })
```

Comment:
because these tests are not async, do we really need to make them async and then await them?

Reply:
You are correct, we don't need to await these. Seems to be the only case/file where this happens. Fixed.